code/modules/jobs/jobs.dm

```diff
@@ -18,25 +22,31 @@ GLOBAL_LIST_INIT(brotherhood_positions, list(
 
 GLOBAL_LIST_INIT(den_positions, list(
 	"Sheriff",
+	"Den Doctor",
 	"Settler"
 ))
 
 GLOBAL_LIST_INIT(legion_positions, list(
-	"Centurion",
-	"Veteran Decanus",
-	"Vexillarius",
-	"Decanus",
-	"Explorer",
-	"Scout",
+	"Legion Centurion",
+	"Legion Veteran Decanus",
 	"Veteran Legionnaire",
-	"Legionary",
+	"Legion Prime Decanus",
+	"Prime Legionnaire",
+	"Legion Recruit Decanus",
+	"Recruit Legionnaire",
+	"Legion Vexillarius",
+	"Legion Explorer",
+	"Legion Scout",
```

Comment:
These are a bunch of changes without changing the actual job titles, and will therefore probably break things. Last time we did something like this people ended up spawning in the arrivals shuttle and other wacky things.

Reply:
I *actually tested it* and it works fine tbh, these are literally just for the manifest/job selection screen